src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11540,9 +11537,9 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         this->PipingCorrectionCooling = TU_CoolingLoad / (TU_CoolingLoad + Pipe_Q_c);
         state.dataHVACVarRefFlow->MaxCoolingCapacity(VRFCond) = this->CoolingCapacity; // for report, maximum evaporating capacity of the system
 
-        this->HeatingCapacity = 0.0;         // Include the piping loss
-        this->PipingCorrectionHeating = 1.0; // 1 means no piping loss
-        state.dataHVACVarRefFlow->MaxHeatingCapacity(VRFCond) = 0.0;
+        this->HeatingCapacity = 0.0;                                 // Include the piping loss
+        this->PipingCorrectionHeating = 1.0;                         // 1 means no piping loss
+        state.dataHVACVarRefFlow->MaxHeatingCapacity(VRFCond) = 0.0; // yujie: might be wrong here too, should be MaxCap = 1e+20
```

Comment:
Please check where MaxHeatingCapacity and MaxCoolingCapacity get set to MaxCap(1.0e+20). That will answer your question. It should either be set to MaxCap or not set at all. Maybe this is the reason that MaxCap is an optional argument, because it's 0 at times. If this var were never set to 0, and only set to MaxCap when reset is needed, and only reduced when the system is overloaded, then MaxCap could always be passed to the coil model and NOT be an optional argument.

Reply:
I just changed MaxCoolCap to non-optional.
I think this initialization here (`state.dataHVACVarRefFlow->MaxHeatingCapacity(VRFCond) = 0.0`) might be just an error. The default for capacity limit should be a really large number, so that no limit is placed.

Reply:
Have you checked where `MaxHeatingCapacity` and `MaxCoolingCapacity` are set to MaxCap(1.0e+20)? Or if you believe this initialization is an error, I think you can correct the code line and delete the comment in this code.

Reply:
It should be 1.0e+20. I'll change it and remove the comment.

Reply:
@dareumnam. I corrected the code line and removed the comment on the 4th. Sorry I forgot to ping you. Please see if it looks okay. Thanks